subsys/bluetooth/host/settings.c

```diff
@@ -233,7 +241,13 @@ static int commit(void)
 {
 	int err;
 
-	BT_DBG("");
+	if (!atomic_test_bit(bt_dev.flags, BT_DEV_ENABLE)) {
+		/* Cannot load settings before bt_enable() because this handler uses bt
+		 * functions. The application shall call settings_load_subtree("bt")
+		 * after bt_enable().
+		 */
+		return 0;
```

Comment:
Ditto regarding the return code

Reply:
As this is no error situation but intended behavior the return code should be 0.